src/acl_kernel_if.cpp

```diff
@@ -1309,6 +1308,8 @@ void acl_kernel_if_launch_kernel_on_custom_sof(
     acl_get_hal()->simulation_streaming_kernel_start(
         kern->physical_device_id,
         kern->streaming_control_signal_names[accel_id]->start);
+    acl_get_hal()->simulation_streaming_interface_done_cra_writes(
+        kern->physical_device_id, accel_id);
```

Comment:
Hmm, yes. I guess the new function only needs to be called whenever `simulation_streaming_kernel_start()` is called. This is not how I had designed it originally, so I had to make a new function. But, now it would make sense to combine the two into the just the `simulation_streaming_kernel_start()` function and change that function's signature to allow for the accel_id.

I'll begin implementing that.